splash/proxy.py

```diff
@@ -65,7 +66,7 @@ def _get_custom_proxy_list(self):
         ]
 
 
-class ProfilesSplashProxyFactory(_BlackWhiteSplashProxyFactory):
+class ProfilesSplashProxyFactory(_AllowDenySplashProxyFactory):
     r"""
     This proxy factory reads BlackWhiteQNetworkProxyFactory
```

Comment:
Could you please also check docstrings?

Reply:
I thought it may be some Qt class, but I see it is a previous name of the parent class.